reactive/easyrsa.py

```diff
@@ -253,10 +261,24 @@ def create_server_certificate(cn, san_list, name='server'):
         cert_file = 'pki/issued/{0}.crt'.format(name)
         # Create the path to the server key.
         key_file = 'pki/private/{0}.key'.format(name)
-        # Do not regenerate the server certificate if it already exists.
-        if not os.path.isfile(cert_file) and not os.path.isfile(key_file):
-            # Get a string compatible with easyrsa for the subject-alt-names.
-            sans = get_sans(san_list)
+        # Create the path to the request file
+        req_file = 'pki/reqs/{0}.req'.format(name)
+        # Get a string compatible with easyrsa for the subject-alt-names.
+        sans = get_sans(san_list)
+        this_cert = {'sans': sans, 'cn': cn, 'name': name}
+        changed = data_changed('server_cert', this_cert)
```

Comment:
Given that easyrsa creates and manages multiple server certs (see [this loop](https://github.com/hyperbolic2346/layer-easyrsa/blob/7fa2ea21b49f3101c5f74d049cc0d48ca9c560ba/reactive/easyrsa.py#L222-L229)), we probably need the data_changed key to be unique for each certificate, yeah? Something like `data_changed('server_cert.' + name, this_cert)`

Reply:
Yes, good catch!